forge-gui/res/cardsfolder/upcoming/navigation_orb.txt

```diff
@@ -0,0 +1,8 @@
+Name:Navigation Orb
+ManaCost:3
+Types:Artifact
+A:AB$ ChangeZone | Cost$ 2 T Sac<1/CARDNAME> | Origin$ Library | Destination$ Hand | ChangeType$ Land.Basic,Gate | ChangeNum$ 2 | RememberChanged$ True | Reveal$ True | Shuffle$ False | StackDescription$ SpellDescription | SubAbility$ DBChangeZone1 | SpellDescription$ Search your library for up to two basic land cards and/or Gate cards, reveal those cards, put one onto the battlefield tapped and the other into your hand, then shuffle.
```

Comment:
I think this should be `Destination$ Library` because of the wording of the card
Line 5 and 6 also use `Origin$ Library`